x-pack/plugins/dashboard_enhanced/public/services/drilldowns/actions/drilldown_shared.ts

```diff
@@ -53,8 +53,8 @@ export const createDrilldownTemplatesFromSiblings = (
   if (!apiIsPresentationContainer(parentApi)) return [];
 
   const templates: DrilldownTemplate[] = [];
-  for (const childId of parentApi.getChildIds()) {
-    const child = parentApi.getChild(childId) as Partial<HasUniqueId & PublishesPanelTitle>;
+  for (const childId of Object.keys(parentApi.children$.value)) {
```

Comment:
Is this going to work? `children$` will not have legacy embeddables so it will return different values than `getChildIds`.